src/tests/test_x509_path.cpp

```diff
@@ -1315,6 +1315,61 @@ class CVE_2020_0601_Tests final : public Test {
 
 BOTAN_REGISTER_TEST("x509", "x509_cve_2020_0601", CVE_2020_0601_Tests);
 
+      #if defined(BOTAN_HAS_PCURVES_BRAINPOOL512R1) || defined(BOTAN_HAS_PCURVES_GENERIC)
```

Comment:
Use `EC_Group::supports_named_group("brainpool512r1")` at runtime

Actually I think you could even run much of the test with the curve absent, we should at least still be able to parse the certs/CRLs.

Reply:
Oh nice. Yeah, that's much better.

I integrated the other test into this one, reverted the `Botan::CHECK` style and instead early returned "successful" if brainpool512 isn't available skipping the path validations.